File: 2956-find-common-elements-between-two-arrays/2956-find-common-elements-between-two-arrays.java
class Solution {
    public int[] findIntersectionValues(int[] nums1, int[] nums2) {
        int c1=0;
        int c2=0;
        int n=nums1.length;
        int m=nums2.length;
        for(int i=0;i<n;i++){
            for(int j=0;j<m;j++){
                if(nums1[i]==nums2[j]){
                    c1++;
                    break;
                }
            }
        }
        for(int k=0;k<m;k++){
            for(int p=0;p<n;p++){
                if(nums2[k]==nums1[p]){
                    c2++;
                    break;
                }
            }
        }
        return new int[]{c1,c2};
        

        
    }
}

File: 2956-find-common-elements-between-two-arrays/2956-find-common-elements-between-two-arrays.py
class Solution(object):
    def findIntersectionValues(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        ans1=[]
        ans2=[]
        count1=0
        count2=0
        for i in nums1:
            if i in nums2:
                count1+=1
        ans1.append(count1)
        for i in nums2:
            if i in nums1:
                count2+=1
        ans2.append(count2)
        list=ans1+ans2
        return list

        